crates/language_model/src/model/cloud_model.rs

```diff
@@ -13,6 +13,7 @@ pub enum CloudModel {
     OpenAi(open_ai::Model),
     Google(google_ai::Model),
     Zed(ZedModel),
+    KimiAi(kimi_ai::Model),
```

Comment:
We don't have any plans to support Kimi AI in our hosted offering, so could you remove anything related to the cloud provider or in `collab` from this PR?